neuralmonkey/model/model_part.py

```diff
@@ -51,7 +51,7 @@ def feed_dict(self, dataset: Dataset, train: bool) -> FeedDict:
     def _init_saver(self) -> None:
         if not self._saver:
             parts_variables = tf.get_collection(
-                tf.GraphKeys.GLOBAL_VARIABLES, scope=self._variable_scope.name)
+                tf.GraphKeys.TRAINABLE_VARIABLES, scope=self._variable_scope.name)
```

Comment:
Mně to přijde, že to má být `MODEL_VARIABLES`. Vizte https://www.tensorflow.org/api_docs/python/tf/GraphKeys